content/docs/userspace/dist/tools.md

```diff
@@ -0,0 +1,372 @@
++++
+title = "Dojo Tools"
+weight = 5
+template = "doc.html"
++++
+
+A number of generators are included for managing and viewing the status of desks and their agents. Each of these are documented below.
+
+Note that some old generators have been replaced:
+
+- `|start` - Replaced with [`|rein`](#rein).
+- `|ota` - Replaced with [`|install`](#install), [`|uninstall`](#uninstall), [`|pause`](#pause) and [`|resume`](#resume).
+- `|fade` - Replaced with [`|nuke`](#nuke).
+- `|doze` - Replaced with [`|rein`](#rein) or [`|suspend`](#suspend).
+- `|wake` - Replaced with [`|rein`](#rein) or [`|revive`](#revive).
+- `+trouble` - This still exists but is deprecated by [`+vats`](#vats).
+
+---
+
+## `+vats`
+
+Print out the status of each installed desk.
+
+Desks in Clay which aren't installed will be omitted.
+
+Fields:
+
+- `/sys/kelvin` - The version of `%zuse` the desk is compatible with.
+- `base hash` - The merge base (common ancestor) between the desk and its upstream source.
+- `%cz hash` - The hash of the desk.
+- `app status` - May be `suspended` or `running`.
+- `force on` - Whether the desk has been force-started.
+- `force off` - Whether the desk has been force-stopped.
```

Comment:
I'm not sure this is correct. I believe these list the apps that have been forced on/off using `|rein`.